### ISSUE TITLE
Multi monitor RDP - enhancement request

### ISSUE THREAD

**Issue by aeolius** [GoogleCloudPlatform/iap-desktop#683]
I realize there is already some multi monitor functionality built in for RDP. But if I'm not mistaken it is based on the span can and has some limitations therein. Would it be possible to update multi monitor functionality to take use of newer mstsc 'selected monitors' setting allowing explicitly setting a contiguous set of monitors to use for RDP and even allowing the choice of which one will be the primary monitor on the remote session?

**Comment by jpassing** [comment]
I agree that "true" multi-monitor support would be better. I tested the `UseMultimon` approach in #352, but encountered another downside:

> [...] Theoretically, these issues could be avoided by using the [UseMultimon](https://docs.microsoft.com/en-us/windows/win32/termserv/imsrdpclientnonscriptable5-usemultimon) feature like mstsc. But the trouble is that this cannot be toggled during a session -- so it breaks the ability to switch back and forth between full-screen and normal screen.

Maybe I missed something... but based on the results of my tests back then, the current approach seemed like the best of two evils.

**Comment by aeolius** [comment]
Looking through the ActiveX Controls myself, not really familiar with them at all, but I don't see any way to toggle UseMultimon after a session has begun so I would guess the only way to make this a possible feature would be to pre-emptively notify the user of this 'issue' ahead of time.
